apps/discovery_streams/lib/discovery-streams/stream/supervisor.ex

```diff
@@ -0,0 +1,19 @@
+defmodule DiscoveryStreams.Stream.Supervisor do
+  @moduledoc """
+  `DynamicSupervisor` implementation. See
+  [Management.Supervisor](../../../../management/lib/management/supervisor.ex)
+  for more details.
+  """
+  use Management.Supervisor, name: __MODULE__
+
+  @impl true
+  def say_my_name(%SmartCity.Dataset{} = dataset) do
```

Comment:
Wait, really? I'm fine with this, but is this a Hindshite thing?

Reply:
it is lol.  Can likely refactor it to have a less silly name but that  wasn't high on my list so far.  It's taken me a long time to get this far just because of the handful of things i didnt understand how they worked at first.

Reply:
Naw, it's good